package.json

```diff
@@ -6,14 +6,17 @@
     "author": "Mubaris",
     "license": "MIT",
     "devDependencies": {
+        "all-contributors-cli": "^4.3.0",
         "codecov": "^2.2.0",
         "eslint": "^3.19.0",
         "eslint-config-airbnb-base": "^11.2.0",
         "eslint-plugin-import": "^2.3.0",
         "mocha": "^3.4.2"
     },
     "scripts": {
-        "lint": "eslint js/**/*.js"
+        "lint": "eslint js/**/*.js",
+        "add": "all-contributors add",
```

Comment:
To make this more clear, can you make the command something like `all-contributors:add` and `all-contributors:generate`?